android/app/src/main/java/updated/mysterium/vpn/model/filter/NodeQuality.kt

```diff
@@ -0,0 +1,16 @@
+package updated.mysterium.vpn.model.filter
+
+import network.mysterium.proposal.QualityLevel
+
+enum class NodeQuality {
+    LOW, MEDIUM, HIGH;
```

Comment:
Enum values should be always on separate lines.

Reply:
Fix